config/routes.rb

```diff
@@ -10,12 +10,21 @@
         post 'delivery'
       end
     end
-    resources :transactions, except: :index
+    resources :transactions, except: :index do
+      member do
+        get 'show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.